php/WP_CLI/CommandWithTranslation.php

```diff
@@ -147,7 +150,8 @@ public function install( $args, $assoc_args ) {
 			\WP_CLI::success( "Language installed." );
 
 			if ( \WP_CLI\Utils\get_flag_value( $assoc_args, 'activate' ) ) {
-				$this->activate( array( $language_code ), array() );
+				$set_date_time = \WP_CLI\Utils\get_flag_value( $assoc_args, 'set-date-time' );
+				$this->activate( array( $language_code ), array( 'set-date-time' => $set_date_time ) );
```

Comment:
Let's only pass `array( 'set-date-time' => $set_date_time )` when it's a truthy value.